.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+on:
+  pull_request:
+    branches:
+    - main
```

Comment:
Just main?

Reply:
Don't we want it for all PRs?

Reply:
I'm going to change it to
```yml
on:
  push:
```
so that it behaves exactly like CircleCI, meaning the workflow will trigger any time a commit is pushed to a branch that has this workflow in it.